src/NServiceBus.Metrics/AllMetrics.cs

```diff
@@ -0,0 +1,67 @@
+namespace NServiceBus.Metrics
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using global::Metrics;
+    using global::Metrics.Core;
+    using global::Metrics.Json;
+    using global::Metrics.MetricData;
+    using global::Metrics.Utils;
+
+    /// <summary>
+    /// Allows to define all avaible metrics
+    /// </summary>
+    public static class AllMetrics
+    {
+        internal static IMetricBuilder[] Create() => new IMetricBuilder[]
```

Comment:
The only thing I don't like about this is that we need to manually keep this list up to date. The previous way you would just drop in a class and it would work.